### PR TITLE
[Recure]: eventName -> eventType

### PR DESCRIPTION
**Pull request by Abnormaltype**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -107,7 +107,7 @@ export async function recure(
   eventName?: string | undefined
 ): Promise<any> {
 
-  if (eventName === EventType.PAGE && !isReadyToSend()) {
+  if (eventType === EventType.PAGE && !isReadyToSend()) {
```
Comment (ivadimko) [inline src/index.ts:110]: what is eventTYpe and eventName?

also
https://mate-academy.github.io/style-guides/javascript.html#710-if-parameters-number-is-greater-than-3-put-them-into-options-parameter

Comment (Abnormaltype) [reply src/index.ts:110]: `eventType` - variable from Enum
```
  LOGIN = "LOGIN",
  SIGN_UP = "SIGN_UP",
  PAGE = "PAGE",
  FREE_TRIAL_STARTED = "FREE_TRIAL_STARTED",
  FREE_TRIAL_ENDED = "FREE_TRIAL_ENDED",
  SUBSCRIPTION_STARTED = "SUBSCRIPTION_STARTED",
  SUBSCRIPTION_ENDED = "SUBSCRIPTION_ENDED"
```

`eventName` - for example `python-name-of-task-started`

Comment (Abnormaltype) [reply src/index.ts:110]: - also
- https://mate-academy.github.io/style-guides/javascript.html#710-if-parameters-number-is-greater-than-3-put-them-into-options-parameter

thanks, will make new changes in the new version later

Comment (IvanRamyk) [reply src/index.ts:110]: why don't make options parameters now? 
also, you wanted to change the name of the cookie, maybe change it now if we will publish a new version anyway?

Comment (ivadimko) [reply src/index.ts:110]: >eventType - variable from Enum
>eventName - for example python-name-of-task-started

it looks super confusing to me

Comment (Abnormaltype) [reply src/index.ts:110]: eventName - is an optional parameter, and using only with eventType PAGE

Comment (ivadimko) [reply src/index.ts:110]: it's better to create an optional parameter `eventOptions` instead
you may need other parameters here further

Comment (Abnormaltype) [reply src/index.ts:110]: did you mean `eventOptions = {eventName: "", otherName: ""}` ?

Comment (ivadimko) [reply src/index.ts:110]: > did you mean `eventOptions = {eventName: "", otherName: ""}` ?

or smth like `{pageName: ""}`
yes

File: src/index.ts [hunk 2]
```diff
@@ -83,16 +87,16 @@ async function getPayload(
...
     visitorId,
     timestamp: new Date().toISOString(),
-    eventName: eventName || "",
+    eventOptions,
   };
 }
 
 function isReadyToSend(): boolean {
-  const readyToSend: string | undefined = Cookies.get("recureReadyToSendEvent");
+  const readyToSend: string | undefined = Cookies.get("recureSendingEventsBlocked");
```
Comment (ivadimko) [inline src/index.ts:95]: better to move cookie name to the constant

**Review by ivadimko — APPROVED** [review]
gj!

**Pull request merged**